src/app/organizations/collection/add-entry/add-entry.component.html

```diff
@@ -0,0 +1,50 @@
+<h1 mat-dialog-title>Add to collection</h1>
+<div mat-dialog-content>
+  <app-alert></app-alert>
+
+  <ng-template #loading>
+    <mat-progress-bar mode="indeterminate"></mat-progress-bar>
+  </ng-template>
+
+  <div *ngIf="!(isLoading$ | async); else loading">
+    <div fxLayout="column" fxLayoutGap="15px">
+      <div fxFlex *ngIf="(memberships$ | async) as memberships">
+        <ng-template #noMemberships>
+          <mat-card class="alert alert-info" role="alert">
+            <mat-icon>info</mat-icon> You must be part of an organization to add to a collection
+          </mat-card>
+        </ng-template>
+        <mat-form-field *ngIf="memberships && memberships.length > 0; else noMemberships">
+          <mat-select id="selectOrganization" placeholder="Organization" [(value)]="selectedOrganisationId" (selectionChange)="onOrganisationChange($event)">
+            <mat-option *ngFor="let membership of memberships" [value]="membership.organisation.id">
+              {{membership.organisation.name}}
+            </mat-option>
+          </mat-select>
+        </mat-form-field>
+      </div>
+
+      <div fxFlex *ngIf="(collections$ | async) as collections">
+        <ng-template #noCollections>
+          <mat-card class="alert alert-info" role="alert">
+            <mat-icon>info</mat-icon> The selected organization has no collections
+          </mat-card>
+        </ng-template>
+        <mat-form-field id="selectCollection" *ngIf="selectedOrganisationId && collections && collections.length > 0; else noCollections">
+          <mat-select placeholder="Collection" [(value)]="selectedCollectionId" [disabled]="!selectedOrganisationId">
+            <mat-option *ngFor="let collection of collections" [value]="collection.id">
+              {{collection.name}}
+            </mat-option>
+          </mat-select>
+        </mat-form-field>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div mat-dialog-actions>
+  <div fxFlex></div>
```

Comment:
Not intuitive, need source.  See src/app/workflow/version-modal/version-modal.component.html for my preferred way of pulling right.